crates/pessimistic-proof/src/multi_batch_header.rs

```diff
@@ -58,6 +58,14 @@ where
     // TODO: benchmark if BTreeMap is the best choice in terms of SP1 cycles
     pub imported_local_exit_roots: BTreeMap<NetworkId, H::Digest>,
 
+    /// The rollup exit root against which we imported the rollup bridge exits
+    #[serde_as(as = "_")]
+    pub imported_rollup_exit_root: H::Digest,
```

Comment:
What does the `#[serde_as(as = "_")]` annotation do here? I briefly looked at the docs but not sure what happens if it's just a wildcard.